python/whylogs/api/writer/whylabs.py

```diff
@@ -603,12 +603,6 @@ def write(self, file: Writable, **kwargs: Any) -> Tuple[bool, str]:
                         f"compared to current datetime: {utc_now}. Uploads of profiles older than 5 years "
                         "might not be monitored in WhyLabs and may take up to 24 hours to show up."
                     )
-                else:
-                    logger.warning(
-                        f"A profile being uploaded to WhyLabs has a dataset_timestamp of {dataset_timestamp} "
-                        f"which is older than 7 days compared to {utc_now}. These profiles should be processed "
-                        f"within 24 hours."
-                    )
```

Comment:
There is some minor cleanup above this removal, we don't need to compare the time_delta_seconds to WEEK_IN_SECONDS anymore, can you switch that to FIVE_YEARS_IN_SECONDS, and if that was the only reference to the WEEK_IN_SECONDS constant we can also remove it?